sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -16,6 +16,7 @@
 
 import com.microsoft.appcenter.AbstractAppCenterService;
 import com.microsoft.appcenter.AppCenter;
+import com.microsoft.appcenter.UserInformation;
```

Comment:
Maybe I didn't explain correctly in previous comments.
Just revert this change.